nginx/nginx.conf

```diff
@@ -2,6 +2,10 @@ upstream frontend {
     server frontend:4200;
 }
 
+upstream backend {
```

Comment:
name of the backend container and upstream clashes here, maybe name upstreams like: `frontend-upstream` and `backend-upstream`? It might be misleading when upstream and container have the same name